test/availability_resolver.test.js

```diff
@@ -0,0 +1,135 @@
+/* eslint-env mocha */
+
+require('dotenv').config()
```

Comment:
This can be removed once https://github.com/NYPL-discovery/discovery-api/pull/39 is merged.